wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -177,17 +177,13 @@ October 25, 2016 (11:45 UTC)
 
 Game modes must be written as follows:
 
-- `osu!standard` (unofficial, but used to prevent ambiguity)
+- `osu!`
```

Comment:
```suggestion
- `osu!` (add `mode` when will be needed)
```

Reply:
``add `mode` when needed``

Reply:
hmm i genuinely don't think that an instruction \*this\* specific is needed tho. i think needless to say all writers by default are already expected to simply apply common sense on a case-by-case basis here whether to write `osu!`, `osu! mode`, `osu! game mode`, or any variations of it in a sentence :(

Reply:
I have already found variates like you describe in the articles, each one has its own way of using the term `osu!` to describe the main game mode, to avoid confusion with the game name.

In the end I would say don't worry too much about it, and formulate it the way you think it should be so that it can be understood.

Without using the term `standard`, because indeed I found it quite misplaced. Indeed we have more chances to confuse the two, but if we talk about spinners or hit objects, we can't go wrong I think.

Honestly, it's really case by case how to name the game mode, we can't define a universal term, because it won't fit well in some cases.

In our case, I think we can leave it as it is, and then I think (with practice), we will know how to use it, maybe it's just a matter of adaptation.